src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -214,6 +218,35 @@ export default class GroupComparisonPage extends React.Component<
                             <MethylationEnrichments store={this.store} />
                         </MSKTab>
                     )}
+                    {this.store.showGenericAssayTab &&
+                        _.map(
```

Comment:
no reason to nest inside of lodash map here.  _.keys returns array so you can just chain _.keys(..).map(..)